### PR TITLE
Update type info for driver

### PR DESCRIPTION
**Pull request by jordanw-bq**
*Issue #, if available:* #129 

*Description of changes:*
- updating type info to report correct information (mostly around literal affixes)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/odfesqlodbc/es_types.c [hunk 1]
```diff
@@ -480,35 +479,35 @@ const char *estype_attr_to_name(const ConnectionClass *conn, OID type,
...
+            return ES_TYPE_NAME_NESTED;
         case ES_TYPE_DATETIME:
-            return "date";
+            return ES_TYPE_NAME_DATE;
         case ES_TYPE_OBJECT:
-            return "object";
+            return ES_TYPE_NAME_OBJECT;
         default:
-            return "unsupported";
+            return "UNSUPPORTED";
```
Comment (kylepbit) [inline src/odfesqlodbc/es_types.c:510]: Believe you're missing the VARCHAR type here.

Comment (jordanw-bq) [reply src/odfesqlodbc/es_types.c]: Added default and case for VARCHAR type (after catching up with Rupal re: a discussion you had with her)

File: src/odfesqlodbc/es_types.c [hunk 2]
```diff
@@ -480,37 +479,36 @@ const char *estype_attr_to_name(const ConnectionClass *conn, OID type,
...
-            return "date";
+            return ES_TYPE_NAME_DATE;
         case ES_TYPE_OBJECT:
-            return "object";
+            return ES_TYPE_NAME_OBJECT;
         case ES_TYPE_VARCHAR:
-            return "varchar";
         default:
-            return "unsupported";
+            return ES_TYPE_NAME_VARCHAR;
```
Comment (rupal-bq) [inline src/odfesqlodbc/es_types.c:511]: IIRC, the default case was correct as unsupported. Separate case for varchar was needed.

Comment (jordanw-bq) [reply src/odfesqlodbc/es_types.c:510]: Added VARCHAR as separate case

**Pull request merged**